source/simulator/assembly.cc

```diff
@@ -1312,14 +1369,21 @@ namespace aspect
                           std_cxx11::_1),
          internal::Assembly::Scratch::
          StokesSystem<dim> (finite_element, mapping, quadrature_formula,
-                            (update_values    | update_gradients |
+                            face_quadrature_formula,
+                            (update_values    |
+                             update_gradients |
```

Comment:
With this change we always have to compute the gradients, instead of only in cases where we rebuild the stokes_matrix. Is this intended/necessary?